docs/source/thanos.rst

```diff
@@ -0,0 +1,83 @@
+Using Thanos as Data Source With Scylla Monitoring
+==================================================
+
+Scylla-Monitoring uses `Prometheus <https://prometheus.io/>`_ for metrics collection, which works out-of-the-box, but Prometheus does have limitations.
+`Thanos <https://thanos.io/>`_  is an opensource solution which when used on top of Prometheus, provides additiuonal functionalities such as:
+
+* High-availability.
+* Horizontal scaling.
+* Backup.
+
+The benefit is that with Thanos' flexible design you can use some or all of these features depending on your requirements.
+
+The rest of this document describes how to place Thanos in front of a multiple Prometheus servers and act as a Grafana datasource instead of Prometheus.
+   
+
+Using Thanos As a Prometheus Aggregator
+---------------------------------------
+There are a few reasons why you would need multiple Prometheus servers: if the total number of your time series reaches millions you can reach the limit of a single Prometheus server capacity.
+Sometimes it is also useful to limit the traffic between data centers, so you can have a Prometheus server per DC.
+
+Prometheus Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+We will assume you have two Prometheus servers running.
+
+1. If you are running Prometheus using a container, you should use an **external** data directory, make sure it is reachable by other containers.
+2. You will need to add the `--web.enable-lifecycle` flag to your Prometheus command-line option.
+
+Thanos sidecar
+^^^^^^^^^^^^^^^
```

Comment:
Thanos side car illustration 
![image](https://user-images.githubusercontent.com/170200/103104916-0076e380-4633-11eb-87dd-7ca6e082c4bd.png)

Reply:
I thought that adding an illustration can make it clearer, I'll use that